docs/General/buildRPC.md

```diff
@@ -45,10 +51,10 @@ Read more about GitHub Actions [here](https://github.blog/2019-08-08-github-acti
 
 **To run the application and connect the managed device:**
 
-1\. On the managed device, run RPC with the following command to activate and configure Intel&reg; AMT. It will take 1-2 minutes to finish provisioning the device.
+1\. Open a Terminal (Linux) or Powershell command prompt (Windows) and navigate to the directory containing the RPC application. Run RPC with the following command to activate and configure Intel&reg; AMT. It will take 1-2 minutes to finish provisioning the device.
```

Comment:
Do we still need a note or sentence here or above reminding folks to download/run on the managed device? Is it intuitive that they would know this?